app/views/admin/administrators/administrators/edit.html.erb

```diff
@@ -14,6 +14,7 @@
             label: { text: "Email" })
       %>
       <%= f.govuk_submit "Save" %>
+      <%= govuk_link_to "Delete User", delete_admin_administrator_path(@administrator), button: true, class: "govuk-button--warning" %>
```

Comment:
Get a design review for this too. Both for button copy and the page copy. They need to be more clear – "Delete user" sounds a bit too vague. Also check if it should be in title case.